src/publicodes-state/helpers/dottedNamesMigration.ts

```diff
@@ -0,0 +1,14 @@
+type migrationType = {
+  key: Record<string, string>
+  value: Record<string, Record<string, string>>
+}
+
+export const dottedNamesMigration: migrationType = {
```

Comment:
Ne vaudrait-il pas mieux ajouter les règles renommées directement dans le paquet NPM ?

Reply:
C'est à dire ?

Là mon idée était vraiment d'avoir une première version d'un fichier de migration pour tester

Reply:
Yes carrément ! Peut-être que dans un premier temps on peut itérer sur le [fichier](https://github.com/incubateur-ademe/nosgestesclimat-site-nextjs/blob/9716b4ce940b2faa478e2701bdb0daa04958d35b/src/constants/dottedNamesMigration.ts) depuis le site ? Ou bien le rendre dispo depuis le modèle ?

Tu penses quoi d'ailleurs de la structure du fichier ?